### PR TITLE
Shoes manual

### PR DESCRIPTION
**Pull request by zoomzam**
### Description

Add examples from shoes manual #461 

### Checklist

- [x] Run tests locally

### CODE REVIEW

**Comment by noahgibbs** [comment]
Nice! I've hit "go" on the test run.

**Comment by noahgibbs** [comment]
Looks like at least the download example failed. I'll look at it later.

**Comment by noahgibbs** [comment]
I'm getting the same failure locally as ci, with shoes_manual_image.

File: examples/legacy/working/shoes_manual/image.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ #!ruby
+ Shoes.app do
+   para "Nice, nice, very nice.  Busy, busy, busy."
+   image "static/avatar.png"
```
Comment (gintama91) [inline examples/legacy/working/shoes_manual/image.rb:4]: `#{DIR}/docs/static/avatar.png`

File: examples/legacy/working/shoes_manual/list_box.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+ #!ruby
```
Comment (gintama91) [inline examples/legacy/working/shoes_manual/list_box.rb:1]: this don't work yet , move to not-working ig

Comment (gintama91) [reply examples/legacy/working/shoes_manual/list_box.rb:1]: hm  weirdly :choose is also not working..

**Comment by noahgibbs** [comment]
This is passing now. Merging.

**Review by noahgibbs — APPROVED** [review]

**Pull request merged**